cmd/snap-confine/mount-support-nvidia.c

```diff
@@ -119,6 +119,39 @@ static const char *nvidia_globs[] = {
 	"libnvoptix.so*",
 	"tls/libnvidia-tls.so*",
 	"vdpau/libvdpau_nvidia.so*",
+
+	// additional libraries for Tegra
+	"libnvdc.so*",
+	"libnvos.so*",
+	"libnvrm_gpu.so*",
+	"libnvimp.so*",
+	"libnvrm.so*",
+	"libnvrm_graphics.so*",
+	"libnppc.so*",
+	"libnppig.so*",
+	"libnppial.so*",
+	"libnppicc.so*",
+	"libnppidei.so*",
+	"libnppist.so*",
+	"libnppcif.so*",
+	"libnppim.so*",
+	"libnppitc.so*",
+	"libcufft.so*",
+	"libcublas.so*",
+	"libcublasLt.so*",
+	"libcusolver.so*",
+	"libcuparse.so*",
+	"libcurand.so*",
+	// libraries for CUDA DNN
```

Comment:
Similarly for this comment if there is a good upstream reference for this

Reply:
none availkable